packages/sampler/src/MoonstoneEnvironment/MoonstoneEnvironment.js

```diff
@@ -73,6 +73,7 @@ const locales = {
 	'th-TH - Thai, with tall characters':                   'th-TH',
 	'ar-SA - Arabic, RTL and standard font':                'ar-SA',
 	'ur-PK - Urdu, RTL and custom Urdu font':               'ur-PK',
+	'zh-Hans-HK - Simplified Chinese, custom Hans font':    'zh-Hans-HK',
```

Comment:
Hongkong(HK) and Taiwan(TW) use Traditional Chinese, China uses Simplified Chinese in webOS.
So, I think that the locale should be `zh-Hans-CN`.

Reply:
Good to know, @seonghyup. Will fix this up.